etc/nginx/snippets/rewrites.conf

```diff
@@ -1,15 +1,16 @@
 # NOTE rewrite uses 302 redirect by default (assuming replacement does not start with a protocol)
 # current versions
-set $current_version_sdk_c_api '3.3.5';
-set $current_version_sdk_dotnet_api '3.4.5';
+set $current_version_sdk_c_api '3.3.7';
+set $current_version_sdk_cxx_api '1.0.0-dp.5';
```

Comment:
do you want non x.y.z versions being published?

Reply:
should be OK, as this is only declaring a variable, not adding the repo to the playbook

Reply:
As it stands, we're not putting docs up before GA, so the situation won't arise (this is a placeholder),
but it's not impossible that beta docs would be needed here or elsewhere.